### PR TITLE
PR

### PR DESCRIPTION
**Pull request by GALLEGOOOO**

### CODE REVIEW

**Review by Manel0712 — CHANGES_REQUESTED** [review]

File: exercise_01/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//test with value as boolean and log result
+console.log("Boolean: " + transformStringToArray(true, onError, onSuccess));
+
+//test with value as empty string and log result
+console.log("Empty String: " + transformStringToArray(" ", onError, onSuccess));
+
+//test with value as string and log result
+console.log(
+  "String: " + transformStringToArray("Em dic Arnau", onError, onSuccess),
+);
```
Comment (Manel0712) [inline exercise_01/index.js:20]: Use the format const = function to execute + console.log(const)

**Review by Manel0712 — CHANGES_REQUESTED** [review]

File: exercise_01/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import onError from "./callbacks/onError.js";
+import onSuccess from "./callbacks/onSuccess.js";
+import transformStringToArray from "./transformStringToCollection.js";
+
+//test with value as number and log result
+console.log("Number: " + transformStringToArray(1, onError, onSuccess));
+
```
Comment (Manel0712) [inline exercise_01/index.js:7]: Use the format const = function to execute + console.log(const)

File: exercise_01/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+//test with value as array and log result
+console.log("Array: " + transformStringToArray(["2", "4"], onError, onSuccess));
+
```
Comment (Manel0712) [inline exercise_01/index.js:10]: Use the format const = function to execute + console.log(const)

File: exercise_01/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+//test with value as boolean and log result
+console.log("Boolean: " + transformStringToArray(true, onError, onSuccess));
+
```
Comment (Manel0712) [inline exercise_01/index.js:13]: Use the format const = function to execute + console.log(const)

File: exercise_01/index.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+//test with value as empty string and log result
+console.log("Empty String: " + transformStringToArray(" ", onError, onSuccess));
+
```
Comment (Manel0712) [inline exercise_01/index.js:16]: Use the format const = function to execute + console.log(const)

**Review by Manel0712 — CHANGES_REQUESTED** [review]

**Review by Manel0712 — APPROVED** [review]

**Review by tictools — APPROVED** [review]

**Pull request merged**